runtime/doc/api.txt

```diff
@@ -1757,18 +1788,18 @@ nvim_buf_get_extmarks({buffer}, {ns_id}, {start}, {end}, {opts})
 
                 Example:
 >
-                  local a   = vim.api
-                  local pos = a.nvim_win_get_cursor(0)
-                  local ns  = a.nvim_create_namespace('my-plugin')
-                  -- Create new extmark at line 1, column 1.
-                  local m1  = a.nvim_buf_set_extmark(0, ns, 0, 0, 0, {})
-                  -- Create new extmark at line 3, column 1.
-                  local m2  = a.nvim_buf_set_extmark(0, ns, 0, 2, 0, {})
-                  -- Get extmarks only from line 3.
-                  local ms  = a.nvim_buf_get_extmarks(0, ns, {2,0}, {2,0}, {})
-                  -- Get all marks in this buffer + namespace.
-                  local all = a.nvim_buf_get_extmarks(0, ns, 0, -1, {})
-                  print(vim.inspect(ms))
+    local a   = vim.api
```

Comment:
@tjdevries this indentation seems like a regression, was it intentional for some reason?